blockchain panic

p2p enable = false，配置关闭p2p

panic: interface conversion: interface {} is *types.Reply, not *types.PeerList

goroutine 308 [running]:
github.com/33cn/chain33/blockchain.(*BlockChain).fetchPeerList(0xc00000a3c0, 0x1034d0f, 0x2101988)
	/home/jiangpeng/go/pkg/mod/github.com/33cn/chain33@v1.65.3-0.20210413063121-cb3cad806da6/blockchain/blocksyn.go:324 +0x833
github.com/33cn/chain33/blockchain.(*BlockChain).FetchPeerList(0xc00000a3c0)
	/home/jiangpeng/go/pkg/mod/github.com/33cn/chain33@v1.65.3-0.20210413063121-cb3cad806da6/blockchain/blocksyn.go:301 +0x70
created by github.com/33cn/chain33/blockchain.(*BlockChain).SynRoutine
	/home/jiangpeng/go/pkg/mod/github.com/33cn/chain33@v1.65.3-0.20210413063121-cb3cad806da6/blockchain/blocksyn.go:179 +0x537

--- a/blockchain/blocksyn.go
+++ b/blockchain/blocksyn.go
@@ -321,8 +321,8 @@ func (chain *BlockChain) fetchPeerList() error {
 		return err
 	}
 
-	peerlist := resp.GetData().(*types.PeerList)
-	if peerlist == nil {
+	peerlist, ok := resp.GetData().(*types.PeerList)
+	if !ok {
 		synlog.Error("fetchPeerList", "peerlist", "is nil")
 		return types.ErrNoPeer
 	}

